### DISCUSSION TITLE
How to keep cursor in highlighted line and clear query?

### DISCUSSION THREAD

**Discussion by sarim** [junegunn/fzf#3443] [Q&A] [answered]
I want to type a word for example "mango", fzf will show results matching "mango", Now I want to press a key, this'll clear the query so all results are being shown now, but I want the highlighted / selected line to stay so my cursor is in "mango" line, and I can see the before / after lines of "mango", and I can up-down arrow to navigate around here.

What I tried:
```
fzf --bind 'tab:clear-query'
```

This clears query, but returns marker to initial position :(

**Comment by LangLangBart** [comment, marked as answer]
- `--track`  flag
  - added with [0.39.0](https://github.com/junegunn/fzf/blob/master/CHANGELOG.md#0390) 
```sh
echo {A..D}{1..4} | tr ' ' '\n' | fzf --query 'B1' --track
```

- `track`  action
  - added with [0.40.0](https://github.com/junegunn/fzf/blob/master/CHANGELOG.md#0400)
```sh
echo {A..D}{1..4} | tr ' ' '\n' | fzf --query 'B1' --bind 'tab:track+clear-query'
```

**Reply by sarim** [reply]
Thank you very much :D `--bind 'tab:track+clear-query'` does exactly this. I was messing with `expect` then reading input again and filtering input to 10 lines around matched line etc..... I only needed one flag.

**Comment by junegunn** [comment]
See #4534
